README.md

```diff
@@ -45,13 +45,27 @@ Follow the few steps described below to start using Unified InstantSearch E-Comm
 2. [**Fork this GitHub repository**](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) to your own,
 3. **Replace the values in `src/config/index.js`** to better match your needs,
 4. **Run `npm run export` (or `yarn export`)** to build the JS and CSS files,
-5. **Host and include the generated JS and CSS files** on your front-end and start using!
+5. **Host and include the generated JS and CSS files** on your front-end:
+
+```html
+<html>
+  <head>
+    <link href="search.css" rel="stylesheet" />
+    <script src="search.js" defer></script>
+  </head>
+
+  <body>
+    <div id="search-button"></div>
```

Comment:
This way it's 100% clear what this is and refers to the associated config. WDYT?

```suggestion
     <!-- The `inputContainer` where to inject the search box -->
    <div id="search-button"></div>
```